tools/slang-unit-test/unit-test-vtable-stability.cpp

```
@@ -0,0 +1,1655 @@
+    {
+        lastSlot = 2;
+        return 1;
+    }
+    SLANG_NO_THROW SlangResult SLANG_MCALL precompileForTarget(SlangCompileTarget, ISlangBlob**)
+        SLANG_OVERRIDE
+    {
+        lastSlot = 3;
+        return SLANG_OK;
+    }
+    SLANG_NO_THROW SlangResult SLANG_MCALL
+    getPrecompiledTargetCode(SlangCompileTarget, IBlob**, IBlob**) SLANG_OVERRIDE
```

Cppcheck (2.20.0)

[style] The function 'getPrecompiledTargetCode' is never used.

(unusedFunction)